system-tests/ft-suite/src/test/kotlin/com/hotels/styx/admin/OriginsFileCompatibilitySpec.kt

```diff
@@ -484,8 +534,30 @@ class OriginsFileCompatibilitySpec : FunSpec() {
                             }
                 }
             }
+
+            test("Health checking service no longer appears in the /admin/service/providers endpoint") {
```

Comment:
Probably we don't need /admin/service/providers (and /admin/service/provider/appB below and above) in the test name but something a bit more meaningful for the reader.

Reply:
OK